examples/analog_hamiltonian_simulation/00_Introduction_of_Analog_Hamiltonian_Simulation_with_Rydberg_Atoms.ipynb

```diff
@@ -9,9 +9,13 @@
     "\n",
```

Comment:
<p>Did we generate the figures with the laser beams on our own? Are there any copy right issues?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
The image is directly from Pedro from QuEra, and he gave permission to use it. Do we need this in writing, or is this enough?

Reply:
I've just added a license, see last commit. @christianbmadsen , do you have any more suggestions?